ads/alp/handler.js

```diff
@@ -226,7 +226,7 @@ export function getA2AAncestor(win) {
     return null;
   }
   const amp = origins[origins.length - 2];
-  if (!ampOrigins.hasOwnProperty(amp)) {
+  if (!ampOrigins[amp] && !ampOrigins.hasOwnProperty(amp)) {
```

Comment:
Maybe worth a comment here? As it's to work around a weird behavior of closure compiler.